tests/ui/layout/nullable-pointer-optimization.rs

```diff
@@ -1,27 +1,34 @@
-//@ run-pass
+//! Nullable pointer optimization with iota-reduction for enums.
```

Comment:
The term should be "null pointer optimization", not "nullable"; because types like `&T` and `Box<T>` are not nullable.

Also for the file name